network/client.go

```diff
@@ -14,12 +14,7 @@ func init() {
 	RegisterPacketType(&StatusRet{})
 }
 
-// Client is an interface which is a simpler version of a Router. The main use
-// for a Client is to directly Send something and get a result back. It is used
-// intensively by Services to have a easy external API.
-// Two implementations are done: TcpClient to use for applications and
-// deployement,and localClient to use for local testing alongside with
-// LocalRouter.
+// Client is the external APIs used for services.
```

Comment:
You wrote it's an interface - which it's not. And I didn't see the two implementations, which makes sense, as it's not an interface. So I supposed it was a comment from an earlier version that was an interface and was implemented twice.